aws_common/src/sdk_utils/auth/service_credentials_provider.cpp

```diff
@@ -66,7 +65,7 @@ static bool SetCurlOpt(CURL * curl, CURLoption opt, T lvalue);
 static bool AppendHeader(struct curl_slist ** headers, const char * name, const char * value);
 static bool IsIotConfigValid(const IotRoleConfig & config);
 static bool ParseConfigVale(std::string & value, Aws::String & result);
-static bool ParseConfigValue(std::string & value, long & result);
+static bool ParseConfigValue(std::string & value, int & result);
```

Comment:
Why?
Can we use int32_t instead?

Reply:
The [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Integer_Types) discourages the use of `short`, `long`, `long long`, etc. Using `int` is acceptable.

Sure, using `int32_t` is a good idea.

Reply:
Done.